README.md

```diff
@@ -93,10 +93,10 @@ Table of contents:
 > Angular is a development platform for building mobile and desktop web applications.
 
 #### Official Resources
-* [Site](https://angular.io/)
-* [Blog](http://angularjs.blogspot.com/)
-* [Documentation](https://angular.io/docs/js/latest/)
-* [Cheatsheet](https://angular.io/cheatsheet)
+* [Site](undefined)
+* [Blog](http://angularjs.blogspot.co.il/)
```

Comment:
I think the url should be `.com`, you get the redirected depending on the locale anyway, we just need to add it to the urls to be ignored when tested.